ts/features/fims/history/store/reducer/index.ts

```diff
@@ -18,9 +18,13 @@ const reducer = (
 ): FimsHistoryState => {
   switch (action.type) {
     case getType(fimsHistoryGet.request):
-      return {
-        consentsList: pot.toLoading(state.consentsList)
-      };
+      return action.payload.isFirstRequest
```

Comment:
it can work, but I prefer keeping the flow more explicit;
especially because if I removed the `Pot.noneLoading` cast that would require me to clear the store on screen exit, and I believe that would generate both more code and more "moving parts" than it removes.